chain_tests.json

```diff
@@ -9,6 +9,14 @@
       "eldest": "second_eldest"
     },
 
+    "test_max_sig_chain": {
+      "_comment": "This is Max Krohn's real live sigchain, as of 10 Aug 2017. The immediate reason for including it is the following interesting case: Max has revoked his very first link, which is both 1) a Twitter proof but also 2) implicitly delegates his eldest key. We should not treat this revocation as revoking the eldest key, and if we do, validating his chain will break.",
+      "input": "max_chain.json",
+      "len": 313,
```

Comment:
Without the `IsDirectlyRevoked` tweak in client, this number would have to be 310.